### ISSUE TITLE
500 error on multiple section required=false images

### ISSUE THREAD

**Issue by employee451** [vapid/vapid#167]
**Describe the bug**
When saving a multiple section record with an image field that has `required=false` set, a 500 "Internal Server Error" is returned.

**To Reproduce**
1. Create a new site: `vapid new testsite`
2. Add the following somewhere on `www/index.html`:

```
{{#section customers multiple=true}}
  {{image type=image required=false}}
{{/section}}
```

3. Start the server `vapid start testsite`
4. Go to `localhost:3000/dashboard` and create a login
5. Click on the "Customers" section and save the record without adding an image to the "Image field"
6. The error will appear

**Expected behavior**
To have the ability to refrain from adding an image to a multiple section when the image field has `required=false` set.

**Screenshots**
This is a screenshot of the error that appears:
![image](https://user-images.githubusercontent.com/22524988/72679455-92837d00-3aaf-11ea-9a08-f1902179b90a.png)

**Your Environment**
 - OS: macOS Mojave 10.14.6
 - Browser: Chrome Version 79.0.3945.117 (Official Build) (64-bit)
 - Node Version: 12.14.1 (LTS)
 - Vapid Version: 0.9.3

**Comment by srobbin** [comment]
@employee451 Thanks for letting me know about this. Just published v0.9.4 with a fix. To update:

```
npm install -g @vapid/cli
```

**Comment by employee451** [comment]
Great, thanks a bunch for this
